integration/okhttp3/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile project(':library')
```

Comment:
I think we missed this, see #941
Library module in v3 is not published, everything there depends on `:glide` as far as I remember.